src/call_context.rs

```diff
@@ -107,8 +107,7 @@ impl<'a> CallContext<'a> {
 
         let instance: Instance;
 
-        if let Some(module) = self.state.modules().get_module_ref(&target).get()
-        {
+        if let Some(module) = self.state.get_module_ref(&target).get() {
```

Comment:
I don't think this is part of the scope of this PR, it's not a changed needed.

Reply:
Resolved